packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -0,0 +1,89 @@
+/*!
```

Comment:
I think you need to merge main into this branch to make this delta (and likely also a conflict) go away.

Since our PR workflow does squash merges, git doesn't sort this out on its own.